libs/eo/shared/atomic-design/ui-atoms/src/lib/eo-beta-message/eo-beta-message.component.html

```diff
@@ -0,0 +1,40 @@
+<!--
+@license
+Copyright 2020 Energinet DataHub A/S
+
+Licensed under the Apache License, Version 2.0 (the "License2");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<div class="watt-card watt-space-stack-l" *ngIf="isVisible">
```

Comment:
It is new that this "beta message box" should be able to close. Where does that requirement come from?

Furthermore, closing the box does not permanently remove the box for the user; next time the page is loaded, the box will show again.

Reply:
It wasn't a requirement, I just thought it would be a nice QOL change. The thought was that the closing of the message box, would persist, during the rest of the user session, and then re appear when logging in again. I can either remove the button entirely or implement the functionality for keeping the message box closed during a user session. 

Which do you prefer?